web/src/list_widget.js

```diff
@@ -131,8 +131,13 @@ function is_scroll_position_for_render(scroll_container) {
     );
 }
 
-export function toggle_empty_list_message($container) {
-    const empty_list_message = $container.data("empty");
+export function toggle_empty_list_message($container, filter_value) {
```

Comment:
I think `show_empty_results_message` is a better name for `filter_value`.

Reply:
I'm not sure about that. I think `filter_value` simply represents what it contains.